serenata_toolbox/federal_senate/federal_senate_dataset.py

```diff
@@ -0,0 +1,100 @@
+import os.path
+from urllib.request import urlretrieve
+import numpy as np
+import pandas as pd
+
+from datetime import date
+
+class FederalSenateDataset:
+    URL = 'http://www.senado.gov.br/transparencia/LAI/verba/{}.csv'
+    FIRST_YEAR = 2008
+    NEXT_YEAR = date.today().year + 1
+
+    YEAR_RANGE = range(self.FIRST_YEAR, self.NEXT_YEAR)
+
+    def __init__(self, path):
+        self.path = path
+
+    def fetch(self):
+        for year in YEAR_RANGE:
+            url = self.URL.format(year)
+            file_path = os.path.join(self.path, 'federal-senate-{}.csv'.format(year))
+            urlretrieve(url, file_path)
```

Comment:
I would like to be able feel protect by tests even if I'm offline, so mock urlretrieve will help with it and let us improve the tests in some others aspects.